src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -1029,4 +1031,20 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
       window.scrollTo({ top: y, behavior: 'smooth' });
     }, 0);
   }
+
+  topFunction(): void {
+    window.scrollTo({ top: 0, behavior: 'smooth' });
+  }
```

Comment:
Just a slight naming issue here, perhaps it's clearer to use a more descriptive name like `scrollToTop()`.